index.html

```diff
@@ -30,6 +30,18 @@ <h3>Hobbies</h3>
           <li>electronics</li>
         </ul>
       </div>
+      <div class="person-container" id="kayambi">
+        <h2>Musafiri Kayambi</h2>
+        <p>Ready to learn from the best while contributing on different projects</p>
+        <h3>Hobbies</h3>
+        <ul>
+          <li>Film and Photography</li>
+          <li> Hiking and running</li>
+          <li> Gardening</li>
+          <li>Cooking</li>
+         
```

Comment:
This is fine. If you want you can remove line 42, or you can add more lis and styles if you want to practice more

Reply:
Yap, will do. Thanks for the suggestion.